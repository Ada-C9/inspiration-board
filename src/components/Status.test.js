// import React from 'react';
// import Status from './Status';
// import { shallow } from 'enzyme';
//
//
// describe(`Status`, ()=> {
//   test(`matches snapshot `, () => {
//     const petCollection = mount(
//       <PetCollection updatedStatuscallback={()=>{}}/>
//     );
//
//     expect(petCollection).toMatchSnapshot();
//
//     petCollection.unmount();
//   });
//
//
//   test(`shallow mount`, () => {
//     const petCollection = shallow(
//       <PetCollection updatedStatuscallback={()=>{}} />
//     );
//
//     expect(petCollection).toMatchSnapshot();
//   });
// });
