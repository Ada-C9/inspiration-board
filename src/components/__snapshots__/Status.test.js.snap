// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status matches snapshot with error type 1`] = `
<section
  className="validation-errors-display error"
>
  There was a problem :
  test status
</section>
`;

exports[`Status matches snapshot with success type 1`] = `
<section
  className="validation-errors-display success"
>
  test status - success
</section>
`;
